liquid_node/docker.py

```diff
@@ -10,33 +10,47 @@ def containers(self, labels=[]):
         out = run(f'docker ps -q {label_args}')
         return out.split()
 
-    def exec_command(self, name, *args, tty=False):
+    def exec_command(self, *args, tty=False):
```

Comment:
How about we keep the `name` argument here, and also pass `stdin` and `tty` as boolean arguments? If they are set, you add `-i` and `-t` respectively, instead of `-it` as it is now. I'd avoid parsing the command line as you're doing at line 26 here.